frontend-student/src/components/coursePage/QuizSection.jsx

```
@@ -0,0 +1,204 @@
+                  );
+                  return (
+                    <div
+                      key={`option-${questionItem.id}-${optIndex}`}
+                      className={`flex items-center p-3 border border-gray-200 rounded-md hover:border-blue-300 transition cursor-pointer ${
+                        quizResults && isCorrect ? 'bg-green-100' : 'bg-white'
+                      }`}
+                      onClick={() =>
+                        !quizResults &&
+                        handleOptionSelect(questionItem.id, optionText)
+                      }
+                    >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)